javascript-functions/main.js

```diff
@@ -0,0 +1,22 @@
+function convertMinutesToSeconds(minutes) {
+  var result = minutes * 60;
+  return console.log('convertMinutesToSeconds result:', result);
+}
+
+function greet(name) {
+  var container = 'Hey, ' + name;
+  return console.log('greet result:', container);
+}
+
+function getArea(width, height) {
+  var product = width * height;
+  return console.log('getArea result:', product);
+}
+
+function getFirstName(person) {
+  return console.log('getFirstName result:', person.firstName);
+}
+
+function getLastElement(array) {
+  return array.splice(array.length - 1);
```

Comment:
Oh no!!! This will remove the item from the array entirely. Now whenever this function gets called to see what the last item is, that item ceases to exist. You should just look up the value, not remove it from the array.